src/Microsoft.AspNetCore.Mvc.Razor/RazorViewEngineOptionsSetup.cs

```diff
@@ -10,38 +10,35 @@
 
 namespace Microsoft.AspNetCore.Mvc
 {
+    [Obsolete("This class will be replaced with one that implements IConfigureOptions.")]
```

Comment:
`[Obsolete("This type is for internal use and will be removed in a future version.")]`